content/docs/pipelines/installation/standalone-deployment.md

```diff
@@ -1,201 +1,218 @@
 +++
-title = "Pipelines Standalone Deployment"
+title = "Kubeflow Pipelines Standalone Deployment"
 description = "Instructions to deploy Kubeflow Pipelines standalone to a cluster"
 weight = 20
 +++
 
-As an alternative to [deploying Kubeflow](/docs/started/getting-started/#installing-kubeflow) as a
-whole with many components including pipelines, you also have a choice to deploy
-only Kubeflow Pipelines. Follow the instructions below to deploy
-Kubeflow Pipelines standalone using the supplied kustomize
-manifests.
+As an alternative to deploying Kubeflow Pipelines (KFP) as part of the
+[Kubeflow deployment](/docs/started/getting-started/#installing-kubeflow), you also have a choice
+to deploy only Kubeflow Pipelines. Follow the instructions below to deploy
+Kubeflow Pipelines standalone using the supplied kustomize manifests.
 
-Knowledge about [Kubernetes](https://kubernetes.io/docs/home/), [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) and [kustomize](https://kustomize.io/) will help you understand this
-document better and be able to customize your deployment based on your needs.
+You should be familiar with [Kubernetes](https://kubernetes.io/docs/home/),
+[kubectl](https://kubernetes.io/docs/reference/kubectl/overview/), and [kustomize](https://kustomize.io/).
 
-{{% alert title="Installation outside GCP is also available" color="info" %}}
-<p>This guide currently describes how to install Kubeflow Pipelines standalone 
-on Google Cloud Platform (GCP). You can also install Kubeflow Pipelines on other
-platforms. This guide needs updating. See
-<a href="https://github.com/kubeflow/website/issues/1253">issue #1253</a>.</p>
+{{% alert title="Installation options for Kubeflow Pipelines standalone" color="info" %}}
+This guide currently describes how to install Kubeflow Pipelines standalone
+on Google Cloud Platform (GCP). You can also install Kubeflow Pipelines standalone on other
+platforms. This guide needs updating. See [Issue 1253](https://github.com/kubeflow/website/issues/1253).
 {{% /alert %}}
 
-## Common prerequisites
+## Prerequisites
 
```

Comment:
I would recommend adding a line that describes the purpose of the Prerequisites. 

The Google Developer Docs style guide recommends against [empty headings](https://developers.google.com/style/headings)

Reply:
Thanks for the tip.